README.md

```diff
@@ -83,50 +83,22 @@ spark.cdm.autocorrect.mismatch                    false|true
 
-# Migrating or Validating specific partition ranges
-- You can also use the tool to Migrate or Validate specific partition ranges by using a partition-file with the name `./<keyspacename>.<tablename>_partitions.csv` in the below format in the current folder as input
-```
--507900353496146534,-107285462027022883
--506781526266485690,1506166634797362039
-2637884402540451982,4638499294009575633
-798869613692279889,8699484505161403540
-```
-Each line above represents a partition-range (`min,max`). Alternatively, you can also pass the partition-file via command-line param as shown below
-
-```
-./spark-submit --properties-file cdm.properties \
- --conf spark.cdm.schema.origin.keyspaceTable="<keyspacename>.<tablename>" \
- --conf spark.cdm.tokenrange.partitionFile.input="/<path-to-file>/<csv-input-filename>" \
- --master "local[*]" --driver-memory 25G --executor-memory 25G \
- --class com.datastax.cdm.job.<Migrate|DiffData> cassandra-data-migrator-4.x.x.jar &> logfile_name_$(date +%Y%m%d_%H_%M).txt
-```
-This mode is specifically useful to processes a subset of partition-ranges that may have failed during a previous run.
+# Rerun (previously incomplete) Migration or Validation 
+- You can rerun a Migration or Validation job to complete a previous run that could have stopped for any reasons. This mode will skip any token-ranges from previous run that were migrated or validation successfully. This is done by passing the `spark.cdm.trackRun.previousRunId` param as shown below
 
-A file named `./<keyspacename>.<tablename>_partitions.csv` is auto-generated by the Migration & Validation jobs in the above format containing any failed partition ranges. No file is created if there are no failed partitions. This file can be used as an input to process any failed partition in a following run. You can also specify a different output file using the `spark.cdm.tokenrange.partitionFile.output` option.
 ```
 ./spark-submit --properties-file cdm.properties \
  --conf spark.cdm.schema.origin.keyspaceTable="<keyspacename>.<tablename>" \
- --conf spark.cdm.tokenrange.partitionFile.input="/<path-to-file>/<csv-input-filename>" \
- --conf spark.cdm.tokenrange.partitionFile.output="/<path-to-file>/<csv-output-filename>" \
+ --conf spark.cdm.trackRun.previousRunId=<prev_run_id> \
  --master "local[*]" --driver-memory 25G --executor-memory 25G \
  --class com.datastax.cdm.job.<Migrate|DiffData> cassandra-data-migrator-4.x.x.jar &> logfile_name_$(date +%Y%m%d_%H_%M).txt
 ```
-
-For the Data-Validation step, use the conf option `-conf spark.cdm.tokenrange.partitionFile.appendOnDiff` as shown below. This allows the partition range to be outputted whenever there are differences, not just fails.
-```
-./spark-submit --properties-file cdm.properties \
- --conf spark.cdm.schema.origin.keyspaceTable="<keyspacename>.<tablename>" \
- --conf spark.cdm.tokenrange.partitionFile.input="/<path-to-file>/<csv-input-filename>" \
```

Comment:
should these 2 properties exists now that this is deprecated but still exists?

Reply:
I have removed any `spark.cdm.tokenrange.partitionFile` oriented properties from the README (actual feature is still there but we should remove it soon), do you see any?